MaterialLibrary/src/main/java/io/doist/material/elevation/CompatElevationDrawable.java

```diff
@@ -337,7 +337,7 @@ private void drawShadow(Canvas canvas) {
      */
     private void update(boolean force, boolean async) {
         View view = mViewRef.get();
-        if (view != null) {
```

Comment:
Is it because we bail out [here](https://github.com/Doist/AndroidMaterial/blob/master/MaterialLibrary/src/main/java/io/doist/material/elevation/CompatElevationDrawable.java#L257)?